components/preparing_workspace/preparing_workspace.tsx

```diff
@@ -168,6 +170,7 @@ export default function PreparingWorkspace(props: Props) {
     const [submitError, setSubmitError] = useState<string | null>(null);
     useEffect(() => {
         showOnMountTimeout.current = setTimeout(() => setShowFirstPage(true), 40);
+        props.actions.getProfiles(0, General.PROFILE_CHUNK_SIZE, {roles: General.SYSTEM_ADMIN_ROLE});
```

Comment:
Good questions! I also assume it will be very unlikely to have 101 system admin profiles in any system. The service does support sorting by creation. However, most uses of the selectors don't read the data directly from the server, but from redux, which doesn't retain the sort order since we store it in redux as an object.